website/src/app/kb/authenticate/google/readme.mdx

```diff
@@ -1,27 +1,25 @@
 import Alert from "@/components/DocsAlert";
-import SupportOptions from "@/components/SupportOptions";
 import PlanBadge from "@/components/PlanBadge";
 import Image from "next/image";
+import Link from "next/link";
 
-<PlanBadge plans={["enterprise"]} />
+<PlanBadge plans={["enterprise"]}>
 
 # SSO with Google Workspace
 
-Firezone supports authenticating users to Google Workspace using our custom
-Google Workspace connector. Use this guide if you're looking to setup SSO with
-Google Workspace for your Firezone Enterprise account.
-
-<Alert
-  color="info"
-  html={`
-    Google Workspace integration is available in the Enterprise plan.
-    To authenticate users to your Google Workspace account in the Starter
-    plan <strong>without</strong> automatic user/group sync, use our
-    <a href="/kb/authenticate/oidc" class="text-accent-500 hover:underline">
-      universal OIDC connector
-    </a> instead.
-  `}
-/>
+</PlanBadge>
+
+Firezone integrates with [Google Workspace](https://workspace.google.com) using
+a custom connector that supports both authenticating users and syncing users and
```

Comment:
```suggestion
a direct API integration that supports both authenticating users and syncing users, org units and
```

Reply:
I call these `connectors` in the remainder of this page and other places of the docs, so going to leave it as-is.

Reply:
Maybe `API connector`